src/Button/index.js

```diff
@@ -28,21 +30,94 @@ class Button extends React.PureComponent {
 	};
 
 	static defaultProps = {
-		disabled: false
+		disabled: false,
+		onHoldCallStart: 300,
+		onHoldCallStep: 50,
```

Comment:
```suggestion
		holdStart: 300,
		holdStep: 50,
```
Don't use 'on*' when it's not an event handler (even if it's related)